project/backend/project-manager/docker-compose.yml

```
@@ -2,42 +1,24 @@
+      # Point to the SHARED collector using its container name
       - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4318
       - OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
       - OTEL_SERVICE_NAME=project-manager
+      # Database connection (assuming DB is accessible on host machine's port 3306)
       - DB_URL=jdbc:mysql://host.docker.internal:3306/metadocs
       - DB_USERNAME=root
       - DB_PASSWORD=1234
-    depends_on:
-      - otel-collector
+    networks:
+      - monitoring_net # Connect to the shared network
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)